plugins/modules/file_servers/backup.py

```diff
@@ -117,8 +117,8 @@ def main():
         agent = client.agents.get('File System')
         backupset = agent.backupsets.get(agent.backupsets.default_backup_set if not module.params.get('backupset') else module.params.get('backupset'))
         subclient = backupset.subclients.get(backupset.subclients.default_subclient if not module.params.get('subclient') else module.params.get('subclient'))
-        backup_level = module.params.get('backup_level')
-        if( backup_level):
+        backup_level = module.params.get('backup_level', 'incremental')
+        if backup_level:
```

Comment:
since we introduced this variable that will have incremental by default we can use it directly and we don't need this if-else condition
we can just do backup = subclient.backup(backup_level=backup_level) for all cases